test/raft/raft_address_map_test.cc

```diff
@@ -214,21 +188,22 @@ SEASTAR_THREAD_TEST_CASE(test_raft_address_map_replication) {
         BOOST_CHECK(!m.find(id1));
 
         // Expiring entries are not replicated
```

Comment:
This test has no meaning now.
Either entries are not replicated: expiring and non-expiring ones.
Previously it had meaning because some entries were replicated, some weren't; so we checked that those that weren't, indeed weren't.

You could update this comment (remove "expiring") or just delete the test altogether.